src/components/MainContent/index.jsx

```diff
@@ -18,7 +18,7 @@ const MainContent = (props) => {
   return (
     <div className={containerClass}>
       <div className="row">
-        <div className="col col-12 offset-md-1 col-md-10 ">
+        <div className="col col-12 offset-md-1 col-md-8">
```

Comment:
If you're going to do this, we're going to want an offset of `offset-md-2` so it has 2 empties columns on the left. To match mockup we'd also need to change `.container`'s `max-width` to 1400px, since we're using the Bootstrap 4 `.container` which is 1920px wide currently (since it looks better on large screens).

Reply:
fixed